kiwi/oci_tools/buildah.py

```diff
@@ -161,7 +161,12 @@ def sync_rootfs(self, root_dir, exclude_list=None):
         self._sync_data(
             ''.join([root_dir, os.sep]), self.oci_root_dir,
             exclude_list=exclude_list,
-            options=Defaults.get_sync_options() + ['--delete']
+            options=Defaults.get_sync_options() + [
+                '--filter="-x! user.*"',
```

Comment:
```suggestion
                '--filter', '-x! user.*',
```

… and so on.

Otherwise I get

```
1[ ERROR   ]: 23:08:52 | KiwiCommandError: rsync: stderr: Unknown filter rule: `"-x! user.*"'
```

Reply:
Hey, thanks for giving this a shot. I adapted the PR as you suggested

Reply:
Thanks! it fixes my issue :)